si/orange-widget-base.yaml

```diff
@@ -0,0 +1,1506 @@
+            both: false
+            '%s (%s)': false
+    class `ParameterMapping`:
+        def `_default_values`:
+            '{} is not supported': false
+        def `_default_get_value`:
+            '{} is not supported': false
+        def `_default_set_value`:
+            '{} is not supported': false
+    def `open_widget_classes`:
+        __init_subclass__: false
+    def `override_default_settings`:
+        wb: false
+    __main__: false
+tests/utils.py:
+    def `excepthook_catch`:
+        'The sys.excepthook that was installed by ': false
+        '''excepthook_catch'' context at enter is not ': false
+        the one present at exit.: false
+    class `simulate`:
+        def `combobox_activate_item`:
+            '{!r} not in {}': false
+utils/PDFExporter.py:
+    class `PDFExporter`:
+        def `export`:
+            antialias: false
+            background: false
+            painter: false
+utils/__init__.py:
+    def `to_html`:
+        <=: false
+        '&#8804;': false
+        '>=': false
+        '&#8805;': false
+        <: false
+        '&#60;': false
+        '>': false
+        '&#62;': false
+        =\\=: false
+        '&#8800;': false
+    def `dumpObjectTree`:
+        '{indent}{type} "{name}"': false
+        ' ': false
+    class `DeprecatedSignal`:
+        def `__init__`:
+            Deprecated: false
+utils/cache.py:
+    class `LRUCache`:
+        __dict: false
+        __maxlen: false
+        __miss: false
+        __hit: false
+utils/combobox.py:
+    class `ComboBox`:
+        def `_get_size_hint`:
+            X: true
+    class `_ComboBoxListDelegate`:
+        def `paint`:
+            separator: false
+    class `ComboBoxSearch`:
+        def `_get_size_hint`:
+            X: true
+        def `showPopup`:
+            Filter...: true
+            darwin: false
+utils/concurrent.py:
+    class `FutureRunnable`:
+        def `run`:
+            Exception in worker thread.: false
+    class `FutureWatcher`:
+        def `setFuture`:
+            Future already set: false
+        def `isCancelled`:
+            isCancelled is deprecated: false
+        def `isDone`:
+            isDone is deprecated: false
+        def `result`:
+            Future is not yet done: false
+        def `exception`:
+            Future is not yet done: false
+    class `FutureSetWatcher`:
+        def `__init__`:
+            Future: false
+        def `setFutures`:
+            already set: false
+            __emitpending: false
+            doneAll: false
+        def `flush`:
+            '`flush()` called from a wrong thread.': false
+        def `wait`:
+            Futures were not set.: false
+utils/filedialogs.py:
+    def `fix_extension`:
+        Mismatching extension: Napačna končnica
+        "Extension '{ext}' does not match the chosen file format, {format}.": "Končnica '{ext}' se ne ujema z izbrano obliko datoteke, {format}."
+        \n\nWould you like to fix this?: \n\nPopravim?
```

Comment:
"Naj to popravim?"

Reply:
Zakaj? A ni lepše, kot je? Saj je čisto jasno, kaj bo popravil. ("To" mi zveni okorno.)